### PR TITLE
SAK-47270 SignUp begins/ends dates always appears in english

### PR DESCRIPTION
**Pull request by jesusmmp**
https://sakaiproject.atlassian.net/browse/SAK-47270

### CODE REVIEW

File: signup/tool/src/webapp/js/signupScript.js [hunk 1]
```diff
@@ -762,8 +762,24 @@ function displayDateTime(d){
 	var curr_date = d.getDate();	
 	var curr_month = d.getMonth();
 	var curr_year = d.getFullYear();	
-	return (curr_hour + ":" + curr_min + " " + a_p + ", " + d_names[curr_day] + ", "
+
+	var lang_es = document.querySelector("html").getAttribute("lang");
+
+	if (lang_es == 'es-ES'){
```
Comment (bjones86) [inline signup/tool/src/webapp/js/signupScript.js:768]: While this provides support for es-ES, what about all other non-default locales? This approach doesn't seem ideal to provide real i18n support.

Comment (ottenhoff) [reply signup/tool/src/webapp/js/signupScript.js:768]: portal.locale contains the logged-in user's locale

date.toLocaleDateString
date.toLocaleTimeString

look like they can accomplish what you need

Comment (jesusmmp) [reply signup/tool/src/webapp/js/signupScript.js:768]: Thanks @ottenhoff! I'll take a look.

Comment (jesusmmp) [reply signup/tool/src/webapp/js/signupScript.js]: @bjones86 @ottenhoff Fixed!

**Review by juanjmerono — APPROVED** [review]

**Pull request merged**